src/main/java/sirius/biz/tenants/UserAccountSearchProvider.java

```diff
@@ -101,7 +101,7 @@ public void query(String query, int maxResults, Consumer<OpenSearchResult> resul
                                                               @user (@user.getTenant().fetchValue().toString())
                                                               <br>
                                                               <a href="/user-accounts/select/@user.getIdAsString()" class="card-link">@i18n("TenantController.select")</a>
-                                                              <i:if test="UserContext.get().getUser().hasPermission('flag-system-support')">
+                                                              <i:if test="UserContext.get().getUser().hasPermission('permission-select-tenant')">
```

Comment:
Please use the Java constant `TenantUserManager.PERMISSION_SELECT_TENANT` instead of the hard-coded string in the templates. You will find examples of if using PERMISSION_SELECT_USER_ACCOUNT for instance.